README.md

```diff
@@ -56,35 +57,47 @@ namespace rootfinding {
 
 The implementation would look like
```

Comment:
This sentence was for `<<algorithm>>` code block, now the codeblock is the equation we want to find the root for.
Please replace sentence with equation intro.

Reply:
done